patches/minecraft/net/minecraft/block/Block.java.patch

```diff
@@ -191,24 +181,24 @@
      }
  
      protected ItemStack func_180643_i(IBlockState p_180643_1_)
-@@ -789,6 +812,8 @@
+@@ -789,6 +811,8 @@
          p_176216_2_.field_70181_x = 0.0D;
      }
  
-+    @Nullable
-+    @Deprecated // Forge: Use more sensitive version below: getPickBlock
++    /** @deprecated by Forge: Use more sensitive version below: {@link #getPickBlock(IBlockState, RayTraceResult, World, BlockPos, EntityPlayer)} */
++    @Deprecated
```

Comment:
We do not apply links or anything in comments because these get obfed to SRG names and can change.
So for 1/2 the users the link will be invalid people will complain.
If you DO add links in the patches it must be in SRG name.
However, problem wiht doing that is it quickly gets patched by our setup. So again, don't do links.